internal/cmd/commands/daemon/start.go

```diff
@@ -130,7 +130,8 @@ func (c *StartCommand) AutocompleteFlags() complete.Flags {
 
 func (c *StartCommand) Run(args []string) int {
 	const op = "daemon.(StartCommand).Run"
-	ctx := c.Context
+	ctx, cancel := context.WithCancel(c.Context)
```

Comment:
Rather than create another context here, can the handler just take `c.ContextCancel` and call it?